### PR TITLE
Change stuns per minute

### PR DESCRIPTION
**Pull request by Argn0**
It just occurred to me, that the value probably should be per minute.

### CODE REVIEW

File: util/benchmarksUtil.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const benchmarks = {
     return p.tower_damage;
   },
   stuns(m, p) {
```
Comment (howardchung) [inline util/benchmarksUtil.js]: do we want to change this too?

Comment (Argn0) [reply util/benchmarksUtil.js]: I dont think so. Because there is only a limited amount of tower damage you can do per game.

Comment (howardchung) [reply util/benchmarksUtil.js]: oh, i was referring to the key name for stuns.  github doesn't highlight the line very well

Comment (Argn0) [reply util/benchmarksUtil.js]: Oh yea, my bad actually.

**Pull request merged**